crowdsec-docs/docs/profiles/format.md

```diff
@@ -131,6 +131,16 @@ on_failure: break
 
 If the profile didn't apply and `on_failure` is set to `break`, decisions processing will stop here and it won't evaluate against following profiles.
 
+### `on_error`
+
+```yaml
+on_error: continue
```

Comment:
enumerate the possible values (apply/continue/break/ignore) and make the default more explicit please